Cargo.lock

```diff
@@ -451,7 +451,7 @@ dependencies = [
 [[package]]
 name = "beefy-gadget"
 version = "4.0.0-dev"
-source = "git+https://github.com/paritytech/substrate?branch=master#26193945883c773f2f14f976da3c3bd11dc0eb79"
+source = "git+https://github.com/paritytech/substrate?branch=master#7de3d7870f63a525977fd4e91cda16039e7b1d60"
```

Comment:
Maybe you should rollback `Cargo.lock` and let the bot eventually update it when your Substrate PR is merged

Reply:
I merged master, and rolled back changes in `Cargo.lock`. I also triggered CI on substrate repo.